### PR TITLE
feat:  update dependency and support new api

### PR DESCRIPTION
**Pull request by flywukong**
### Description
1) support group new API, including  "group ls ", "group ls-member", "group ls-belong"
2) update depenency
3) support "policy ls" command

### Rationale

meet new requirement

### Example

```
 qa-test % ./gnfd-cmd group ls

create-time            group-name           id
2023-08-18 12:45:33    skkl1                12
2023-08-18 13:06:23    test-groupxj1        13
2023-08-18 13:07:09    test-groupxj2        14
2023-08-18 13:08:29    test-groupxj3        15
2023-08-18 13:08:36    test-groupxj4        16
2023-08-18 13:08:43    test-groupxj5        17

a-test % ./gnfd-cmd group ls-member skkl1
create-time         member                                     expire-time
2023-08-18 13:05:18 0x4eca7038852B11e27A8CAAAD30bB3609087D458b 10000-01-01 07:59:59
2023-08-18 13:13:40 0xADff5FF66bB6Cf77dcdd087ad54a0838f7C5e520 10000-01-01 07:59:59
2023-08-18 13:25:27 0xadFF5fF66Bb6Cf77dCDd087Ad54a0838f7C5e522 10000-01-01 07:59:59
2023-08-18 13:25:39 0xADff5fF66bb6cF77dCdd087ad54a0838f7c5e523 10000-01-01 07:59:59
2023-08-18 13:26:11 0xadfF5Ff66bB6CF77DcDD087aD54A0838f7C5e524 10000-01-01 07:59:59
2023-08-18 18:13:03 0xaDff5fF66BB6cF77DCdD087AD54A0838F7C5e525 2023-08-18 18:18:36

 qa-test % ./gnfd-cmd group ls-belong --address 
Please enter a passphrase now:
2023-08-18 13:05:18                          skkl1 id: 12
2023-08-18 13:14:43                  test-groupxj2 id: 14
2023-08-18 13:14:55                  test-groupxj3 id: 15

 qa-test % ./gnfd-cmd  policy ls --grantee 0xf95061C7cc0A2A0B52a6c6a7f6a192c824480423   grn:o::dskl1/testv1
principal                                            actions                                  effect     resource
grantee:0xf95061C7cc0A2A0B52a6c6a7f6a192c824480423   GET_OBJECT,DELETE_OBJECT                 ALLOW      gnfd://dskl1/testv1

qa-test % ./gnfd-cmd --passwordfile password.txt policy ls --grantee 0xf95061C7cc0A2A0B52a6c6a7f6a192c824480423 grn:g:klklll:skkl1
principal                                            actions                                  effect     resource
grantee:0xf95061C7cc0A2A0B52a6c6a7f6a192c824480423   UPDATE_GROUP_MEMBER                      ALLOW      group:skkl1
```

### Changes

Notable changes:
* add each change in a bullet point here
* ...

### CODE REVIEW

File: cmd/cmd_group.go [hunk 1]
```diff
@@ -86,19 +87,94 @@ Examples:
...
+
+// cmdListGroup is the command to list members info of the particular group.
+func cmdListGroup() *cli.Command {
+	return &cli.Command{
+		Name:      "ls",
+		Action:    listGroup,
+		Usage:     "list  groups owned by the specified user",
+		ArgsUsage: "GROUP-NAME",
+		Description: `
+ Returns a list of groups owned by the specified user,
```
Comment (BarryTong65) [inline cmd/cmd_group.go:144]: delete the extra space before returns

File: cmd/cmd_group.go [hunk 2]
```diff
@@ -315,6 +400,157 @@ func renewGroupMember(ctx *cli.Context) error {
 	return nil
 }
 
+// listGroupMember returns a list of members contained within the group specified by the group id
+// The user's expiration time has already elapsed will not be listed
```
Comment (BarryTong65) [inline cmd/cmd_group.go:411]: the expired user will be listed

File: cmd/cmd_group.go [hunk 3]
```diff
@@ -315,6 +400,157 @@ func renewGroupMember(ctx *cli.Context) error {
...
+	for {
+		memberList, err := client.ListGroupMembers(c, int64(groupInfo.Id.Uint64()),
+			sdktypes.GroupMembersPaginationOptions{Limit: maxListMemberNum, StartAfter: initStartKey})
+		if err != nil {
+			return toCmdErr(err)
+		}
+
+		printListMemberResult(memberList)
+		memberNum := len(memberList.Groups)
+		if memberNum != maxListMemberNum {
```
Comment (BarryTong65) [inline cmd/cmd_group.go:446]: if memberNum <  maxListMemberNum will be more reasonable

File: cmd/cmd_group.go [hunk 4]
```diff
@@ -315,6 +400,157 @@ func renewGroupMember(ctx *cli.Context) error {
...
+	for {
+		groupList, err := client.ListGroupsByOwner(c,
+			sdktypes.GroupsOwnerPaginationOptions{Limit: maxListMemberNum, Owner: groupAccount, StartAfter: initStartKey})
+		if err != nil {
+			return toCmdErr(err)
+		}
+
+		printListGroupResult(groupList)
+		memberNum := len(groupList.Groups)
+		if memberNum != maxListMemberNum {
```
Comment (BarryTong65) [inline cmd/cmd_group.go:492]: if memberNum <  maxListMemberNum will be more reasonable

**Review by BarryTong65 — APPROVED** [review]

**Review by alexgao001 — APPROVED** [review]

**Pull request merged**